WeakAurasOptions/ConditionOptions.lua

```diff
@@ -876,7 +876,7 @@ local function addControlsForChange(args, order, data, conditionVariable, condit
           end
         end
       end
-      return false
+      return needle == nil
```

Comment:
This is pretty strange, what are you trying to do here?

Reply:
this is to make anyGlowExternal("glow_frame_type", nil) return true if glow_frame_type is nil

if glow_frame_type is nil then
`local ref = type(conditions[i].changes[j].value) == "table" and conditions[i].changes[j].value[property]`
will be nil and the function will always return false without this change